platform/mellanox/one-image.mk

```diff
@@ -1,6 +1,7 @@
 # sonic mellanox one image installer
 
 SONIC_ONE_IMAGE = sonic-mellanox.bin
+$(SONIC_ONE_IMAGE)_DEPENDS += $(SYSTEMD_SONIC_GENERATOR)
```

Comment:
i do not think DEPENDS work, add to INSTALLS

Reply:
I think both work (at least for the virtual switch). Do we prefer using INSTALLS over DEPENDS?

Reply:
it works because you manually installed the package in the sonic_debian_extension.j2, but if you add to _INSTALLS then you do not need to manually install the package. the package will be automatically installed.

Reply:
DEPENDS is the build dependency, it install package in the sonic-slave. INSTALL installs the package in the sonic image.

Reply:
Ok I understand now